client/scripts/views/components/sidebar/index.ts

```diff
@@ -594,8 +594,18 @@ const Sidebar: m.Component<{ hideQuickSwitcher? }, {}> = {
         m('br'),
         app.isLoggedIn() && (app.chain || app.community) && m(SubscriptionButton),
         app.chain && m(ChainStatusModule),
-      ])
-    ];
+        app.isCustomDomain && m(SidebarQuickSwitcher),
+        m('br'),
+        m('br'),
+      ]),
+      app.isCustomDomain(),
```

Comment:
Why is there a hanging `isCustomDomain` here? Not a mithril component -- needs to be `&&` with the PoweredBy link below.

Reply:
Updated and the extra line break is removed